docs/docs/migration-guide.mdx

```diff
@@ -1254,7 +1254,157 @@ consult the following migration guides for the individual policies:
 
 The `ConveRTTokenizer`, `LanguageModelTokenizer`, and `HFTransformersNLP` featurizer
 components were deprecated in Rasa 2.x and have been removed in Rasa 3.0. See the
-[migration guide for Rasa 2.x](#deprecations-2) for replacing these components in your pipeline.  
+[migration guide for Rasa 2.x](#deprecations-2) for replacing these components in your pipeline.
+
+### Slot Mappings
+
+As of version 3.0, there is a single explicit mechanism for slot filling enabled by defining slot mappings for each slot
+in the `slots` section of the domain file. This approach keeps slots up to date over the course of a conversation, and
+removes duplicated effort in mapping the same slots in multiple forms.
+
```

Comment:
Similar to the comment above about `custom` mappings being allowed to be empty: The phrasing here could make a user think they can't define "empty" slots that are filled only at specific points in the conversation anymore. But they can - they can just do more than that. 
```suggestion
As of version 3.0, there is a single explicit mechanism for slot filling enabled by defining slot mappings for each slot
in the `slots` section of the domain file. This approach keeps slots up to date over the course of a conversation, and
removes duplicated effort in mapping the same slots in multiple forms. It is still possible to fill slots from arbitrary custom actions and not update them on every turn of the conversation if that behavior is desired.

```